src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -312,6 +315,15 @@ def _add_endpoint_args(parser):
         help=f"Set a custom endpoint that differs from the OpenAI defaults",
     )
 
+    endpoint_group.add_argument(
+        "--endpoint-type",
```

Comment:
boy scout fix

Reply:
Please don't fix this in this PR. This needs its own discussion. We already have --endpoint. If we're going to do this for ease, then we should have a bigger discussion over whether we remove --endpoint since these two are functionally identical. We'd also need to add documentation and testing.

Reply:
This is just moving it around to adhere to listing the options in alphabetical order per subgroup. This changed no functionality.
If you want this in a separate fix, we can do that but we have previously discusses small "boy scout" changes like this being okay.